contracts/axelar-gateway/src/executable.rs

```diff
@@ -2,7 +2,7 @@ use core::panic;
 
 use soroban_sdk::{contractclient, Address, Bytes, Env, String};
 
-use crate::axelar_gateway::AxelarGatewayClient;
+use crate::contract::AxelarGatewayClient;
 
 /// Interface for an Axelar Executable app.
 #[contractclient(name = "AxelarExecutableClient")]
```

Comment:
export the client from `lib.rs`

Reply:
should I also export `AxelarGateway`? They seem to be imported in a few places (like the `gmp.rs` integration test)

Reply:
yes, should export both clients